packages/SwingSet/src/controller.js

```diff
@@ -68,9 +72,17 @@ export async function makeSwingsetController(
   harden(console);
 
   // FIXME: Put this somewhere better.
-  process.on('unhandledRejection', e =>
-    console.error('UnhandledPromiseRejectionWarning:', e),
-  );
+  const handlers = process.listeners('unhandledRejection');
```

Comment:
This is good, but let's put it in a separate PR.

Reply:
done